charts/mimir-distributed/values.yaml

```diff
@@ -223,9 +223,7 @@ mimir:
 # runtimeConfig provides a reloadable runtime configuration file for some specific configuration.
 runtimeConfig: {}
 
-rbac:
-  create: true
-  pspEnabled: true
+rbacEnabled: true
```

Comment:
is this how loki has it? I recall different, also this would be a breaking change since rbac.pspEnabled would be gone as well, please double check

Reply:
aside from `pspEnabled` option, loki has the extra ` sccEnabled` (SecurityContextConstraints) option, and the rolebinding resource is rendered whenever either one is enabled.

if you consider, i could add this extra option as well. otherwise i think the `rbacEnabled` should be enough.

Reply:
forget my last comment, since as you have rightly pointed out this would mean a breaking change. let me rework this.